engines/bastion_katello/app/assets/stylesheets/bastion_katello/host_subscriptions.scss

```diff
@@ -0,0 +1,16 @@
+table.table-bordered {
+  tbody {
+    tr.row-selector-label {
```

Comment:
Do you need all this specificity to override a class?  Generally you only use the amount of specificity needed to override something.  

If we removed `.table-bordered` here then none of these styles would apply any longer.  That example is somewhat contrived but you can see how fragile extra specificity can be.  I'd recommend removing `.table-bordered` at a minimum.